operator/Dockerfile

```diff
@@ -56,7 +56,7 @@ RUN GOOS=$(go env GOOS) GOARCH=$(go env GOARCH) scripts/go-build.sh operator
 # Copy the operator binary to a location from where it will be taken into the final image.
 RUN cp -a bin/$(go env GOOS)_$(go env GOARCH)/operator stackrox-operator
 
-FROM registry.access.redhat.com/ubi8-micro:8.7
+FROM registry.access.redhat.com/ubi8-micro:latest
```

Comment:
I don't know what to say to this... :/ Downstream does use latest too, but latest is not stable and is subject to continuously change.
But also I don't have knowledge about the ubi8-micro image.

Reply:
Some things to note here:
* latest is stable in this context, RH strongly recommends using latest (e.g. see the message [here])(https://catalog.redhat.com/software/containers/ubi8-micro/601a84aadd19c7786c47c8ea?architecture=amd64&image=6543bfed5a39159da4340be9) and it's the only way to make sure that we don't ship images with vulnerabilities
* it's only the latest release in terms of ubi8 (so it won't do a major upgrade to ubi9)
* the master branch already uses latest (see https://github.com/stackrox/stackrox/pull/6795)
* I could use 8.9 instead (but it would basically point to the same image)